files/HashFiles.java

```diff
@@ -0,0 +1,67 @@
+/*
+ This program and the accompanying materials are made available
+ under the terms of the Eclipse Public License v2.0 which
+ accompanies this distribution, and is available at
+ https://www.eclipse.org/legal/epl-v20.html
+
+ SPDX-License-Identifier: EPL-2.0
+
+ Copyright Contributors to the Zowe Project. 2020
+ */
+
+import java.io.*;
+import java.lang.*;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class HashFiles  // input is a filename; file contains a list of filenames to be hashed
+{
+   
+   public static long RSHash(String str)
+   {
+      int b     = 378551;
+      int a     = 63689;
+      long hash = 0;
+
+      for(int i = 0; i < str.length(); i++)
+      {
+         hash = hash * a + str.charAt(i);
+         a    = a * b;
+      }
+
+      return hash;
+   }
+
+   public static String readAllBytesJava7(String filePath)
+   {
+       String content = "";
+
+       try
+       {
+           content = new String ( Files.readAllBytes( Paths.get(filePath) ) );
+       }
+       catch (IOException e)
+       {
+           e.printStackTrace();
+       }
+
+       return content;
+   }
+   
+   public static void main(String args[]) throws IOException
+   {
+      File file=new File(args[0]);    
+      FileReader fr=new FileReader(file);   
+      BufferedReader br=new BufferedReader(fr);  
+      String line;  
+      while((line=br.readLine())!=null)  
+      {  
```

Comment:
Can you fix the formatting to have correct style with the `{`s not on a separate line.

Reply:
There are many "correct" styles.  I've used the style of the code I originally downloaded.

Reply:
Other zowe projects have a java linter. Your code is not compliant with these.

Reply:
Since we don't have a java linter, I don't know which style to follow.  There are many here https://en.wikipedia.org/wiki/Indentation_style.  The only other java program is `MD5Checksum.java`.  Should I follow that style?

Reply:
I've put the `{` at EOL.

Reply:
I've run it through the explorer-api-common linter and put the result in https://gist.github.com/stevenhorsman/8619765534a40daad8e4b8da81c10728